### PR TITLE
Docker issues

### PR DESCRIPTION
**Pull request by scottwn**
- [x] I have added the tests to cover my changes.
- [x] I have updated the documentation and CHANGELOG accordingly.
- [x] I have read the CONTRIBUTING document.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,63 +1,91 @@
...
   covalent-queue-consumer:
-    image: "public.ecr.aws/covalent/covalent_queuer"
+    build:
+      context: .
+      dockerfile: refactor/queuer/Dockerfile
+    image: covalent-queue-consumer
+    command: sh -c "python main.py"
     ports:
-      - "80:8001"
+      - "3000:8001"
```
Comment (wjcunningham7) [inline docker-compose.yml:52]: does this service need to expose a port?

File: docker-compose.yml [hunk 2]
```diff
@@ -1,63 +1,91 @@
...
   covalent-ui:
-    image: "public.ecr.aws/covalent/covalent_ui"
+    build:
+      context: .
+      dockerfile: refactor/ui_backend/Dockerfile
+    image: covalent-ui
+    command: sh -c "python main.py"
     ports:
-      - "80:8004"
+      - "8005:8004"
```
Comment (wjcunningham7) [inline docker-compose.yml:88]: should this be 8005:8005?

**Review by wjcunningham7 — APPROVED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (scottwn) [reply docker-compose.yml:88]: no, the default port for the UI backend is `8004`....... I think 🙃 
supervisord sets it to `8005`, so docker-compose needs to do the same thing

File: docker-compose.yml (hunk 1 above)
Comment (scottwn) [reply docker-compose.yml:52]: I'm not sure... @AlejandroEsquivel @Emmanuel289 does this look right for the consumer?

Comment (Emmanuel289) [reply docker-compose.yml:52]: @scottwn I exposed port 8001, similar to the queuer. Maybe @AlejandroEsquivel can confirm if this is correct for the service.

Comment (AlejandroEsquivel) [reply docker-compose.yml:52]: @scottwn @Emmanuel289 the queue consumer does not expose an API, it just connects to NATS so we do not require port mappings here

**Review by FyzHsn — APPROVED** [review]

**Pull request merged**